packages/core/directives/map.ts

```diff
@@ -478,6 +496,24 @@ export class AgmMap implements OnChanges, OnInit, OnDestroy {
     this._observableSubscriptions.push(s);
   }
 
+  private _handleDragEvent() {
+    const s = this._mapsWrapper.subscribeToMapEvent<void>('drag').subscribe(
+        () => { this.drag.emit(void 0); });
+    this._observableSubscriptions.push(s);
+  }
+
+  private _handleDragStartEvent() {
+    const s = this._mapsWrapper.subscribeToMapEvent<void>('dragstart').subscribe(
+        () => { this.dragStart.emit(void 0); });
+    this._observableSubscriptions.push(s);
+  }
+
+  private _handleDragEndEvent() {
+    const s = this._mapsWrapper.subscribeToMapEvent<void>('dragend').subscribe(
+        () => { this.dragEnd.emit(void 0); });
+    this._observableSubscriptions.push(s);
+  }
+
   private _handleMapMouseEvents() {
```

Comment:
this would be better to handle this events in this function (so we would have much less code)